targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F429xI/device/TOOLCHAIN_IAR/stm32f429xx_flash.icf

```diff
@@ -17,7 +17,7 @@ define symbol __ICFEDIT_region_CCMRAM_end__   = 0x1000FFFF;
 /*-Sizes-*/
 /*Heap 1/2 of ram and stack 1/8*/
 define symbol __ICFEDIT_size_cstack__ = 0x6000;
-define symbol __ICFEDIT_size_heap__   = 0x18000;
```

Comment:
is it 1/2 of RAM (comment says so) ? Is it 1/4 as thats usual number ?

Let's make sure this new number works for all reported issues we had within last days, and any future failure shall be a known issue?